apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info-tigger.tsx

```diff
@@ -41,16 +40,19 @@ export const AssetTooltip = ({
         title="Options"
         onClick={() => setTooltipOpen(true)}
         css={{
-          opacity: 0,
+          visibility: "var(--info-trigger-visibility)",
```

Comment:
visibility value passed from parent using css variable to avoid nested selectors that may have specificity conflicts later in the future